TripKit/Classes/model/API/AlertAPIModels.swift

```diff
@@ -61,7 +61,15 @@ extension API {
     public let operators: [String]?
     public let serviceTripIDs: [String]?
     public let stopCodes: [String]?
-    public let routeIDs: [String]?
+//    public let routeIDs: [String]?
+    public let routes: [API.Route]?
+    public let modeIdentifier: String?
+  }
+  
+  public struct Route: Codable {
```

Comment:
I'm not sure which of these properties should be "optional". The data from Fernando is currently inconsistent, some `route` JSON is missing `id, some missing `number`, some missing `name`. Given that I'll need to group alerts by route, it'd help if one of these properties is non-optional. Is this something we should get from ITO so not to put more efforts into it?

Reply:
Fer said that he'll make ID non-optional. Number and name will remain optional as we don't get those for every route.